crossing-repair.lic

```diff
@@ -12,7 +12,7 @@ class CrossingRepair
   def initialize
     setup
 
-    ensure_copper_on_hand(@repair_withdrawal_amount, @bank_info)
```

Comment:
Because crossing repair is only going to happen in Crossing and ensure_copper_on_hand defaults to Crossing.